docker-gen.go

```diff
@@ -72,6 +72,7 @@ type RuntimeContainer struct {
 	Volumes      map[string]Volume
 	Node         SwarmNode
 	Labels       map[string]string
+	Server       ServerInfo
```

Comment:
It seems like putting it on `Context` would be quite a bit of effort and wouldn't work that well.

Assuming there would be a `func (c *Context) Docker() *Docker` function, that function would have to call `getEndpoint()` and `NewDockerClient()` to create a Docker client each time it's called since it has no access to an existing `docker.Client`. I also can't see how `docker-gen` would be able to avoid hitting the Docker server every time the function is called.

I suppose you could stash a reference to the `docker.Client` inside of each `RuntimeContainer`, but that seems like a terrible hack.

I could see making it work if `Context` were turned into a `struct`, but then it would not be possible to call `range` on it and iterate over a list of `RuntimeContainer` instances without changing every existing `docker-gen` template.